src/operations/common/logging.js

```diff
@@ -19,10 +19,15 @@ const fhirLogger = require('../../utils/fhirLogger').FhirLogger;
  */
 const setRequestIdInLog = (args) => {
     const reqId = httpContext.get('requestId');
-    if (reqId) {
+    const userRequestId = httpContext.get('userRequestId');
+    // eslint-disable-next-line no-prototype-builtins
+    if (reqId && args && args.hasOwnProperty('request')) {
```

Comment:
`args.hasOwnProperty('request')` should not be required since we want to add the request id even if request object is not sent.